async-nats/src/jetstream/consumer/pull.rs

```diff
@@ -186,7 +186,8 @@ impl Stream for Batch {
 #[derive(Debug, Default, Serialize, Deserialize, Clone, Copy, PartialEq, Eq)]
 pub struct BatchConfig {
     /// The number of messages that are being requested to be delivered.
-    pub batch: usize,
+    #[serde(rename = "batch")]
+    pub limit: usize,
```

Comment:
To be frank, I don't see an advantage to justify such a rename.

I guess you're coming from the idea that the number of data returned can be `limited` by either batch_size or max_bytes?

If yes, this makes it even more confusing without renaming bytes.

`messages_limit` and `bytes_limit` ?

Reply:
Trying to avoid the stutter of `batch(batch)`, limit came to mind because of the iterator 🤔

Reply:
Hmm, max_messages?